.restyled.yaml

```diff
@@ -0,0 +1,12 @@
+restylers:
+  - clang-format:
+	  enabled: false
```

Comment:
I think this red highlighting on GitHub is indicating actual-tabs?

Reply:
Not sure how that got in there.